---
title: BioNetVisA workshop 2018
---

<center><a href="/"><img id="ICSB19" src="/ISCB2019_logo.jpg" style="width:160px;"/></a></center>
<p> </p>

# <center><font color="#e60000">BioNetVisA 2019 workshop</font></center>
## <center><font color="#e60000">From biological network reconstruction to data visualization and analysis in molecular biology and medicine</font></center>
<!--
<p align="center"><b><font color="#e60000">The final programme and the abstract booklet are available</font><a href="/Abstracts/Bionetvisa2018/BioNetVisA2018_booklet.pdf"> here</a></b></p>
<p align="center"><b><font color="#e60000">The workshop presentations are available</font><a href="https://drive.google.com/open?id=1cLa4gSUAeogCNUi1yAFzlidHbOrxakxh"> here</a></b></p> -->

<p align="justify"><b>BioNetVisA</b> workshop brings together different actors of network biology from database providers, networks creators, computational biologists, biotech companies involved in data analysis and modeling to experimental biologists, clinicians that use systems biology approaches. The participants are exposed to the different paradigms of network biology and the latest achievements in the field.
</p>

### Motivation

<p align="justify">The goal of <b>BioNetVisA</b> workshop is to build a discussion around various approaches for biological knowledge formalisation, data integration and analysis; compatibility between different methods and biological networks resources available the field; applicability for concrete research and clinical projects depending on scientific question and type of high-throughput data.
</p>

<p align="justify">The <b>BioNetVisA</b> workshop aims at identifying bottlenecks and proposing short- and long-term objectives for the community as discussing questions about accessibility of available tools for wide range of user in every-day standalone application in biological and clinical labs. In addition, the possibilities for collective efforts by academic researchers, clinicians, biotech companies and future development directions in the field will be discussed during the round table panel.
</p>

### Audience
<p align="justify">The workshop targets computational systems biologists, molecular and cell biologists, clinicians and a wide audience interested in update and discussion around current status of network biology, pathway databases, and related analysis tools, including visualization, statistical analysis and dynamic modelling. No computational background is required to attend the workshop.
</p>

### Key Themes
<p>Graphical representation of biological knowledge<br />
Molecular interaction and pathway databases<br />
Comprehensive signalling networks<br />
Networks annotation and curation<br />
High-throughput data visualization, analysis and interpretation in the context of networks<br />
Multi-scale networks (genome, epigenome, transcriptome, proteome, metabolome...)<br />
Network modelling<br />
Machine learning/Artificial Intelligence approaches in network biology <br />
Basic research and clinical application of networks<br />
Microbiome and networks<br />
Single-cell data and network inference<br />
Metabolomics and networks<br />
Human microbiome<br />
Networks for drug repositioning<br />
</p>

### Important dates
<table style="width 100%">
<tr><td><b>August 1</b></td>
  <td>Abstract submission deadline<b><i><font color="#e60000">Submit now!</font></i></b></td></tr>
<tr><td><b>August 7</b></td>
  <td>Abstract acceptance notification<b></td></tr>
<tr><td><b>August 20</b></td>
  <td>Workshop's final program announcements</td></tr>
<tr><td><b>September 1</b></td>
  <td>Workshop's registration deadline (no fee)</td></tr>
<tr><td><b>October 31</b></td>
  <td>BioNetVisA 2019</td></tr>
</table>

### Preliminary program

<table>
<tr><td><strong><font color="#e60000">Session 1: Pathway resources and maps construction in the AI era</font></strong></td></tr>
<tr><td><b>08:50 - 09:00 Opening</b></td></tr>
<tr><td><b>09:00 - 09:30 <br />
<strong>Deriving structured representations from natural language to aid scientific discovery: Case studies from reconstructing large pathway maps</strong><br />
<a href=""><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a><br />
<a href="https://suchee.bitbucket.io/">Sucheendra Kumar Palaniappan</a></b><br />
<i>The Systems Biology Institute, Tokyo, Japan</i></td></tr>
<tr><td><b>09:30 - 10:00 <br />
<strong>Talks selected from abstracts</strong><br /></td></tr>
</table> 
  
<table>
<tr><td><strong><font color="#e60000">Session 2: AI for network-based drug discovery and repositioning</font></strong></td></tr>
<tr><td><b>10:00 - 10:30 <br />
<strong>DPharma Databases Integration coupled with human curation for Drug Development/Repurposing</strong><br /> 
<a href="/Abstracts/Bionetvisa2019/Georgios_Pavlopoulos_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a>

<table> 
 <tr><td><i>10:30 - 11:00 Coffee break</i></td></tr>
</table>


<tr><td><b>11:00 - 11:25 <a href="http://pavlopouloslab.info">Georgios Pavlopoulos</a></b><br />
<i>Biomedical Sciences Research Center "Alexander Fleming", Athens, Greece</i><br />
<strong>Using HipMCL, a high-performance parallel implementation of the Markov clustering algorithm, to understand microbial diversity</strong><br />
<a href="/Abstracts/Bionetvisa2018/Georgios_Pavlopoulos_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>11:25 - 11:50 <a href="https://www.researchgate.net/profile/Anna_Niarakis">Anna Niarakis</a></b><br />
<i>GenHotel EA3886, Univ Evry, Universit√© Paris-Saclay, France</i><br />
<strong>Disease Networks - Reconstruction, Topology, Dynamics. Towards an automated pipeline from static representations to executable disease models</strong><br />
<a href="/Abstracts/Bionetvisa2018/Anna_Niarakis_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>11:50 - 12: 10 <a href="https://www.research.ed.ac.uk/portal/en/persons/barbara-shih(0a863546-b83e-4884-90cc-4d796da614d4).html">Barbara Shih</a></b><br />
<i>The Roslin Institute and Royal (Dick) School of Veterinary Studies, the University of Edinburgh, Easter Bush, Midlothian, Edinburgh, UK</i><br />
<strong>A platform for the network assembly and visual analysis of transcript isoforms from short-read RNA-sequencing data</strong><br/>
<a href="/Abstracts/Bionetvisa2018/Barbara_Shih_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>12:10 - 12:30 <a href="http://www.warrenwk.com/">Warren W Kretzschmar</a></b><br />
<i>KTH Royal Institute of Technology, Stockholm, Sweden</i><br />
<strong>Multicolored De Bruijn graph visualization of first, second, and third generation RNA-seq offers deeper insight into transcript isoforms in Picea abies</strong><br />
<a href="/Abstracts/Bionetvisa2018/Warren_Kretzschmar_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
</table>

<table>
<tr><td><i>12:30 - 13:30 Lunch</i></td></tr>
</table>

<table>
<tr><th><font color="#e60000">13:30 - 15:00 Session 3: Biological networks in single-cell data analysis</font></th></tr>
<tr><td><b>Chair: <a href="https://lunean.com/">Augustin Luna</a></b></td></tr>
<tr><td><b>13:30 - 13:55 <a href="http://www.vib.be/en/research/scientists/Pages/Yvan-Saeys-Lab.aspx">Yvan Saeys</a></b><a href="http://isac-net.org/"><font color="Red"><b> ISAC Lecture</b></font></a><br />
<i>VIB-Ghent University, Gent, Belgium</i><br />
<strong>Inferring dynamic regulatory networks from single cell data</strong><br />
<a href="/Abstracts/Bionetvisa2018/Yvan_Sayes_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>13:55 - 14:20 <a href="https://sysbio.curie.fr/people.html">Andrei Zinovyev</a></b><br />
<i>Institute Curie, Paris, France</i><br />
<strong>Biological networks help unraveling tumoral heterogeneity at single cell level</strong><br />
<a href="/Abstracts/Bionetvisa2018/Andrei_Zinovyev_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>14:20 - 14:40 <a href="https://www.researchgate.net/profile/Anirudh_Patir3">Anirudh Patir</a></b><br />
<i>The University of Edinburgh, Edinburgh, UK</i><br />
<strong>Evaluation of network methods for the analysis scRNA-seq data and development of a new KNN-based method based on identified caveats</strong><br />
<a href="/Abstracts/Bionetvisa2018/Anirudh_Patir_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>14:40 - 15:00 <a href="https://science.curie.fr/members/vassili-soumelis-md-phd/">Vassili Soumelis</a></b><br />
<i>Institut Curie, Cancer Immunotherapy Center, Paris, France</i><br />
<strong>Immune cell diversification in response to a single stimulus: from single cells to subsets and back</strong><br />
<a href="/Abstracts/Bionetvisa2018/Vassili_Soumelis_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
</table>

<table>
<tr><td><i>15:00 - 15:30 Coffee break</i></td></tr>
</table>

<table>
<tr><th><font color="#e60000">15:30 - 17:00 Session 4: Biological network in drug repositioning and disease comorbidity</font></th></tr>
<tr><td><b>Chair: <a href="http://peoplewiki.clinbioinfosspa.es/jdopazo/">Joaquin Dopazo</a></b></td></tr>
<tr><td><b>15:30 - 16:15 <a href="https://www.bsc.es/valencia-alfonso">Alfonso Valencia</a><font color="red"> Keynote Lecturer</font></b><br />
<i>Barcelona Supercomputing Center, Barcelona, Spain</i><br />
<strong>A patient centred disease comorbidity network</strong></td></tr>
<tr><td><b>16:15 - 16:35 <a href="http://www.ibens.ens.fr/spip.php?article95">Celine Hernandez</a></b><br />
<i>Institut de Biologie de l‚ÄôEcole Normale Sup√©rieure, Paris, France</i><br />
<strong>Dynamical modelling of T cell co-inhibitory pathways to predict anti-tumour responses to checkpoint inhibitors</strong><br />
<a href="/Abstracts/Bionetvisa2018/Celine_Hernandez_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>16:35 - 16:55 <a href="https://www2.unil.ch/cbg/index.php?title=Sven_Bergmann">Sven Bergmann</a></b><br />
<i>University of Lausanne & Swiss Institute of Bioinformatics, Switzerland</i><br />
<strong>Open Community Challenge Reveals Molecular Network Modules with Key Roles in Diseases</strong><br />
<a href="/Abstracts/Bionetvisa2018/Sven_Bergmann_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><b>16:55 - 17:15 <a href="https://www.researchgate.net/profile/Josephine_Daub">Josephine Daub</a></b><br />
<i>Princess M√°xima Center for Pediatric Oncology, Utrecht, The Netherlands</i><br />
<strong>Genetic interactions in childhood cancer</strong><br />
<a href="/Abstracts/Bionetvisa2018/Josephine_Daub_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
</table>

### Topics
<p>Graphical representation of biological knowledge<br />
Pathway and network resources<br />
Networks annotation and curation<br />
Multi-omics data visualization and analysis in the context of networks<br />
Network modelling<br />
Basic research and clinical application of networks<br />
Microbiome and networks<br />
Networks for single-cell data interpretation<br />
Metabolomics and networks<br />
Networks applications for drug repositioning<br />
Networks in disease comorbidity studies<br />
</p>

### Scope
<p align="justify"><b>Development, curation and maintenance of biological network resources</b><br />
Data models from interaction to pathway and network, available tools for the integration of new discoveries into existing signalling diagrams. Discussion of curation models from map manager-oriented to community-based update using web-tools and forums.
</p>

<p align="justify"><b>Cell signalling knowledge representation paradigms</b><br />
From interaction to single pathways representation to comprehensive signalling networks: discussion of modes of biological processes representation; granularity, coverage, sources of information, annotation modes, cross-citations between different signalling networks resources, etc. Standard for graphical representation and data formats.
</p>

<p align="justify"><b>Visualization of data through networks</b><br />
Paradigms of visualization of different types of high-throughput data on a network map. Data integration with a map. Methods compatibility with different paradigms of signalling networks reconstruction in various databases and resources.
</p>

<p align="justify"><b>Analytical tools for network biology</b><br />
Tools for network analysis and manipulation, network modeling, network-based statistics. Studying network activation and inhibition, feedbacks and compensations, synthetic interactions.
</p>

<p align="justify"><b>Applications of networks in multi-omics data analysis in biomedical research and in clinics</b><br />
Studies on single-cell data, microbiome, metabolomics and other omics data analysis, disease
comorbidity, drug response prediction using network-based approaches.
</p>

<!--
### Abstract submission instructions
We invite to submit your abstract for a <b>talk</b> in the topics described above.
#### <font color="#e60000">The submission deadline is August 10th 2018</font>
<p>Submitted abstracts will be reviewed by the scientific program committee and a notification of acceptance for a <b>talk</b> will be provided to the corresponding author by August 12th, 2018.
</p>
<p><b>Abstract format:</b><br />
Title<br />
List of authors (the first author is the presenting author)<br />
Affiliations<br />
Abstract text in unstructured format (maximum 300 words)
</p>
<p align="justify"><b>Submission process</b><br />
The link below will take you to the EasyChair submission page. Once logged, click on the 'Submission' tab to start the submission process. Enter authors, title without HTML elements, abstract up to 300 words, keywords, and select up to <b><span style="text-decoration: underline;">three</span></b> relevant topic sessions.
</p>
<p align="justify">To assist us with creating the abstract booklet, please upload the same abstract as a word document in the section called "Upload Paper" using this <b><a href="/AbstractTemplate_BioNetVisA.docx"><span style="text-decoration: underline;"><font color="#e60000">TEMPLATE</font></span></a></b>
</p>
<p align="justify">When the form is adequately filled out, press the 'Submit' button. You will receive an email from the conference "EasyChair". This email is solely a notification that EasyChair for BioNetVisA2014 has received the abstract.
</p>
<b><a href="https://easychair.org/conferences/?conf=bionetvisa2018"><font size="5" color="#e60000"><center>Submit your abstract here</center></font></a></b> -->

### Registration

<p>The registration has to be performed via the <b><a href="http://eccb18.org/registration/"><font color="#e60000">ECCB website</font></a></b>
</p>

### Organising committee
<p><a href="http://sysbio.curie.fr/">Inna Kuperstein</a> (Institut Curie, France)<br />
<a href="https://science.curie.fr/recherche/biologie-interactive-des-tumeurs-immunologie-environnement/c/">Emmanuel Barillot</a> (Institut Curie, France)<br />
<a href="http://www.ihes.fr/~zinovyev/">Andrei Zinovyev</a> (Institut Curie, France)<br />
<a href="http://sysbio.curie.fr/">Luis Cristobal Monraz Gomez</a> (Institut Curie, France)<br />
<a href="http://www.sbi.jp/">Hioraki Kitano</a> (RIKEN Center for Integrative Medical Sciences, Japan)<br />
<a href="http://www.kanehisa.jp/en/kanehisa.html">Minoru Kanehisa</a> (Institute for Chemical Research, Kyoto University, Japan)<br />
<a href="http://www.sbi.jp/members.htm">Samik Ghosh</a> (Systems Biology Institute, Tokyo, Japan)<br />
<a href="https://reactome.org/">Robin Haw</a> (Ontario Institute for Cancer Research, Canada)<br />
<a href="https://www.bsc.es/valencia-alfonso">Alfonso Valencia</a> (Spanish National Bioinformatics Institute, Madrid, Spain)<br/>
<a href="https://www.nki.nl/divisions/molecular-carcinogenesis/wessels-l-group/">Lodewyk Wessels</a> (Netherlands Cancer Institute, Amsterdam, Netherlands)<br />
  <a href="https://www.prinsesmaximacentrum.com/">Patrick Kemmeren</a> (Princess Maxima Center for Pediatric Oncology, Utrecht, Netherlands)
</p>

### Sponsors
  <div>
      <a href="http://isac-net.org/" target="_blank"><img src="thumbnail_Logo_ISAC_sponsor.jpg" alt="INCa" style="float: left; width:30%; margin-right:10%; margin-bottom: 0.5em;"></a>
      </div>	
      
<!--
<tr><td><b>09:00 - 09:20 <a href="http://www.sbi.jp/members.htm">Samik Ghosh</a></b><br />
<i>The Systems Biology Institute, Tokyo, Japan</i><br />
<strong>Towards a network-centric approach in systems toxicology</strong><br />
<a href="/Abstracts/Bionetvisa2018/Samik_Ghosh_BioNetVisA2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> -->


